content/knowledge-base/lifecycle.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Product Lifecycle
+---
+
+## Software List
+
+Upbound provides support and software maintenance to paid customers. Supported software products include:
+
+* [UXP](https://www.upbound.io/product/universal-crossplane)
+* [Official Providers](https://marketplace.upbound.io/providers?tier=official)
+
+## Maintenance and Updates
+
+Upbound supports Generally Available releases for up to 12 months. Eligible code-fixes are provided via a new minor release on top of the latest major release branch for up to 3 releases from the most current major release. All software follows semantic versioning of `X.Y.Z`. A major release is a change in the first or second digit. A minor release is a change in the third digit.
+
+### UXP Schedule
+
+UXP follows the [Crossplane Release Cycle](https://docs.crossplane.io/knowledge-base/guides/release-cycle/). UXP has a major release every 3 months. Minor patch release are available as needed. Upbound provides support for the current + 3 previous major releases, for a minimum of 12 calendar months of coverage for a given release.
+
+### Official Provider Schedule
+
+Official providers are released on an as-needed schedule. Historically this has mapped to major releases twice per month. 
```

Comment:
Official providers are released on at least a monthly cadence. 
Technically we start preparing a release on the last Thursday of a month with the goal of releasing on the following Monday at the latest.

I am comfortable committing to a monthly release cadence for providers, with patch releases made available as needed in between major/minor releases.